src/backend/distributed/citus--7.3-3--7.3-4.sql

```diff
@@ -0,0 +1,78 @@
+/* citus--7.3-3--7.3-4 */
+
+/* helper functions for automated failure testing */
+
+CREATE FUNCTION citus.run_command(text)
+RETURNS void AS $$
+  use strict;
+  use warnings;
+  use IO::Socket;
+
+  my $command = $_[0];
+  my $sockpath = '/home/brian/Work/citus/src/test/regress/gdb.sock';
```

Comment:
can you make this a relative path ?

Reply:
I think eventually it'll need to be a hostname:port tuple, otherwise this won't work on Windows.